.github/ISSUE_TEMPLATE/new-validation-rule.md

```diff
@@ -0,0 +1,17 @@
+---
+name: New validation rule
+about: Suggest a new GTFS validation rule for this project
+title: ''
+labels: new rule
+assignees: ''
+
+---
+
+**What problem in GTFS datasets does this new rule address? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
```suggestion
<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --> 
```

Reply:
@lionel-nj I actually think this makes it much harder to read when you're creating the issue, especially if it's your first time creating an issue:

![image](https://user-images.githubusercontent.com/928045/111687073-3fff3700-8800-11eb-8699-6f7b742f2feb.png)

I'd be inclined to err on the side of making it easier for people to read when they are writing the issue - I think we'll get better information. People should really just be deleting this text anyway and replacing it with their own as it's just a placeholder - that's what I normally do, and what I've seen happen on most GitHub repositories that I work with.

What do you think?

Reply:
Alright! Let's stick to what is most commonly done then!